docs/subsystems/2fa.md

```diff
@@ -0,0 +1,13 @@
+# Two Factor Authentication
+
+Zulip uses [django-two-factor-auth][0] to integrate 2FA.
+
+To enable 2FA, set `TWO_FACTOR_AUTHENCATION_ENABLED` in settings to `True`, then
+log into Zulip and add otp device from settings page. Once the device is added,
+password based authentication will ask for one-time-password. In dev., this
+password will be printed to the console.
+
+Direct dev. logins don't prompt for 2FA tokens, so to test 2FA in development,
+make sure that you login using a password.
```

Comment:
Add a note that you can get the passwords for the default test users using `manage.py initial_password`.